src/components/modules/TrackCard/components/PlaceholderTrackCard.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Card,
+  Controls,
+  PlayPauseButton,
+  TrackInfo,
+  TrackTitle,
+  TrackDetails,
+  TrackTags,
+  ProgressContainer,
+  ProgressBar,
+  TrackImg,
+} from './TrackCard.styled'
+import PlaySvg from '../../../../assets/svg/PlaySvg'
+import CardBtns from '../../CardBtn/components/CardBtns'
+import type { Track } from '@/types'
+
+interface Props {
+  data: Track
+  handleClick?: () => void
+}
+
+const PlaceholderTrackCard: React.FC<Props> = ({ data, handleClick }) => {
+  const { id, title, artist, album, genres, coverImage, slug, audioFile } = data
+  const playPointer = audioFile ? 'pointer' : 'not-allowed'
+
+  return (
+    <Card data-testid={'track-item-' + id}>
```

Comment:
```suggestion
    <Card data-testid={`track-item-${id}`}>
```
It's better to use template strings instead of concatenation. Fix in all places